src/app/datagrid/selection/selection.html

```diff
@@ -52,11 +72,9 @@
     <clr-dg-column>Favorite color</clr-dg-column>
 
     <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
-        <clr-dg-action-overflow>
-            <ul class="list-unstyled">
-                <li><button type="submit" class="btn btn-sm btn-link">Edit</button></li>
-                <li><button type="submit" class="btn btn-sm btn-link">Delete</button></li>
-            </ul>
+        <clr-dg-action-overflow [clrDgActionItem]="user">
+            <a class="action-item" href="#" (click)="onEdit()" clrDgActionOverflowToggle>Edit</a>
+            <a class="action-item" href="#" (click)="onDelete()" clrDgActionOverflowToggle>Delete</a>
```

Comment:
If I understand correctly, `clrDgActionOverflowToggle` is here to close the action overflow menu when one of the actions is clicked. While I do approve this behavior, the implementation right now bothers me, it requires a bit too much from the consumer for my taste.

How about just closing on any click on the document except for clicks on the action menu _itself_? Right now it closes only on clicks outside, but by not going up the tree and only checking if the target is the menu itself, any clicks on links inside the menu would still close it.